Revise Cycle 5 funding request guidelines (#5)
eteq/astropy-project: finance/proposal-calls/cycle5/cycle5.md

Money can be used for travel, subscriptions, etc, in addition to direct financial support to individuals, including individuals based outside of the United States.

> Funding requests may be submitted which include the recruitment and hiring of someone to complete the task. See below: Open Hire Process.

Reviewer: ```suggestion
Funding requests may be submitted even if they include the recruitment and hiring of another person to complete the task. See below: Open Hire Process.
```

The funds currently come from a NASA ROSES grant. NumFocus, the Project’s fiscal sponsor, and the Finance Committee assume responsibility for ensuring that all budgets are consistent with any applicable funding agency rules.

# Funding Request Process Overview

> We envision a two stage FR process to enable the community to discuss and improve ideas, to merge teams with similar ideas, and for the SPOC to ensure as much of the Roadmap is possible is being addressed.

Reviewer: ```suggestion
We envision a two stage FR process to enable the community to discuss and improve ideas, to merge teams with similar ideas, and for the SPOC to ensure as much of the Roadmap as possible is being addressed.
```

* Oct 17, 2025: **_Draft_** FR Deadline
* Three week iteration and discussion period during which draft FRs can be fleshed out and modified. During this time:

>   * the SPOC may submit and/or solicit new proposals that fill gaps not addressed by the initial round of requests. 
>   * the Finance Committee will review budgets and may provide feedback and/or request more information. 

Reviewer 2: ```suggestion
  * The SPOC may submit and/or solicit new proposals that fill gaps not addressed by the initial round of requests. 
  * The Finance Committee will review budgets and may provide feedback and/or request more information. 
```

* TBD + 3wk (Nov 7): Finalized FRs are due, two week Voting period begins
* TBD + 5wk (Nov 24): The SPOC and Finance Committee will work together to select funding requests and allocate budgets. 
* TBD + 7 wk (Dec 8): Announcement of selected FRs. Drafting contracts begins.
* Jan 1, 2026: Work commences

We aim to achieve broad consensus before the vote. Requesters are encouraged to combine ideas and teams, modify, or withdraw FRs during the discussion period, as appropriate. Discussion is open to anyone; it is not restricted to voting members. 

Voting members of the Astropy Project vote on the FRs using thumbs up/down on a dedicated comment. Every voting member of the Astropy Project has equal voting rights.

> The Coordination and Finance Committees will select FRs based on alignment with the Roadmap, ranking by the members, available effort to implement, and available funds.

Reviewer 2: ```suggestion
The Coordination and Finance Committees will select FRs based on the voting results, alignment with the Roadmap, confirmation of available people to implement, and available funds.
```
